composer.json

```diff
@@ -26,28 +26,26 @@
     {
       "type": "composer",
       "url": "https://composer.typo3.org/"
-    },
-    {
-      "type": "git",
-      "url": "https://github.com/svewap/realurl_404_multilingual"
     }
   ],
   "minimum-stability": "dev",
   "require": {
     "php": "^7.0",
-    "typo3/cms": "^8.7",
-    "typo3-ter/realurl": "2.2.1",
-    "typo3-themes/themes": "dev-8-0",
+    "typo3/cms": "~8.7.0",
+    "typo3-themes/themes": "~8.7.0",
+    "typo3-ter/dyncss-less": "~0.7.9",
     "gridelementsteam/gridelements": "dev-master",
-    "apache-solr-for-typo3/solr": "6.1.0",
-    "typo3-ter/dyncss-less": "0.7.9",
-    "clickstorm/go_maps_ext": "2.3.0",
-    "clickstorm/cs_seo": "2.1.2",
-    "typo3-ter/news": "6.0.0",
-    "typo3-ter/pxa-newsletter-subscription": "5.0.2",
-    "typo3-ter/pxa-form-enhancement": "3.0.0",
-    "typo3-ter/pxa-cookie-bar": "1.0.7",
-    "svewap/realurl-404-multilingual": "1.0.9",
-    "typo3-ter/frontend-editing": "1.1.2"
-    }
+    "typo3-ter/realurl": "~2.2.1",
+    "clickstorm/go_maps_ext": "~2.3.0",
+    "clickstorm/cs_seo": "~2.1.2",
+    "typo3-ter/news": "~6.0.0",
+    "typo3-ter/frontend-editing": "~1.1.2"
+  },
+  "suggest": {
```

Comment:
In general, looks good, except I am not sure about `suggest` section. All those extensions are a core part of t3kit, so it is not just suggestion, it is integrated into t3kit deeper (ts configs, templates, etc...)
Thanks! 
@pixelmatseriks @MattiasNilsson ?

Reply:
Ok. That's right for `solr` and `pxa_newsletter_subscription` . For the any other extension, i can not found any source code for that extensions.

Reply:
Those extensions included also in our base t3kit db
https://github.com/t3kit/t3kit_db

Reply:
Yes, you write it.
These extensions are no dependencies to `theme_t3kit` extension.
They are dependencies for a t3kit project with your databse content, but not for the extension itself.

At a closer look, the extension `cs_seo` should also moved to section "suggest".
The configration for that extension are only loaded, if extension is installed.

Reply:
It sounds reasonable to me. We will move those extensions as the main dependencies to `composer.json` in `t3kit` repo, and then we can use it as a suggestion here in `theme_t3kit`.

So would be nice if you fix conflicts and update your PR.

Thanks!

Reply:
Ok, I will move `solr` and `pxa_newsletter_subscription` back to section "required".
To analyse which extension is only needed for an optional feature of the `theme_t3kit` extension, I will open an issue. After this analyse, I will open an new pull request for these extension, if needed.